<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInfo</class>
 <widget class="QWidget" name="EditInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/MainWindow/wolf.png</normaloff>:/MainWindow/wolf.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="thumbnail">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>180</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Video Name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextEdit" name="infoEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="tagsLabel">
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="tagsEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="playlistEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="playlistLabel">
     <property name="text">
      <string>Playlist</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
